examples/configuring/README.md

```diff
@@ -0,0 +1,109 @@
+Configuring GraphDB
+===
+
+This document provides detailed instructions on how to configure GraphDB, 
+including setting properties, secret properties, and additional configurations. 
+
+It covers various methods to manage configuration options, such as using 
+ConfigMaps and Secrets, and highlights some good practices to ensure secure 
+and efficient setup. Additionally, it explains how to set Java arguments 
+and environment variables for GraphDB.
+
+All GraphDB configuration options can be found 
+[here](https://graphdb.ontotext.com/documentation/10.0/configuring-graphdb.html#configuration-properties).
+
+## Properties
+
+This section is used to set environment variables directly from the `values.yaml` file. 
```

Comment:
They are not set as env variables, they are inserted in the default graphdb.properties configmap